AmazonChimeSDKDemo/AmazonChimeSDKDemo/LiveTranscriptionOptionsViewController.swift

```diff
@@ -139,31 +230,153 @@ class LiveTranscriptionOptionsViewController: UIViewController, UITextFieldDeleg
         engineTextField.inputView = enginePickerView
         languageTextField.inputView = languagePickerView
         regionTextField.inputView = regionPickerView
+        partialResultsStabilizationTextField.inputView = partialResultsStabilizationPickerView
+        piiContentIdentificationTextField.inputView = piiContentIdentificationPickerView
+        piiContentRedactionTextField.inputView = piiContentRedactionPickerView
         
         engineSelected = "transcribe"
         languageSelected = "en-US"
         regionSelected = "auto"
+        partialResultsStabilizationOption = 0
+        piiContentIdentificationSelected = ""
+        piiContentRedactionSelected = ""
         
         engineTextField.text = enginesDict[engineSelected]
         languageTextField.text = languagesDict[languageSelected]
         regionTextField.text = regionsDict[regionSelected]
-        
+        partialResultsStabilizationTextField.text = partialResultsStabilizationsList[partialResultsStabilizationOption]
+        piiContentIdentificationTextField.text = piiContentIdentificationDict[piiContentIdentificationSelected]
+        piiContentRedactionTextField.text = piiContentRedactionDict[piiContentRedactionSelected]
+    
         enginePickerView.delegate = self
         languagePickerView.delegate = self
         regionPickerView.delegate = self
         enginePickerView.dataSource = self
         languagePickerView.dataSource = self
         regionPickerView.dataSource = self
+        partialResultsStabilizationPickerView.delegate = self
+        partialResultsStabilizationPickerView.dataSource = self
+        piiContentIdentificationPickerView.delegate = self
+        piiContentIdentificationPickerView.dataSource = self
+        piiContentRedactionPickerView.delegate = self
+        piiContentRedactionPickerView.dataSource = self
         
         enginePickerView.tag = 1
         languagePickerView.tag = 2
         regionPickerView.tag = 3
+        partialResultsStabilizationPickerView.tag = 4
+        piiContentIdentificationPickerView.tag = 5
+        piiContentRedactionPickerView.tag = 6
+        
+        storyBoard = UIStoryboard(name: "Main", bundle: nil)
+
+        guard let languageOptionsViewController =
+                storyBoard?.instantiateViewController(withIdentifier: "languageOptions")
+                as? AutomaticLanguageOptionsViewController else { return }
+        self.languageOptionsViewController = languageOptionsViewController
+        
+        languageOptionsViewController.cancellationHandler = { numberOfselectedLanguages in
+            self.enableAutomaticLanguageIdentificationSwitch.isOn = numberOfselectedLanguages! > 1
```

Comment:
nit: the closures between line278-296 should use `weak self` to prevent from retain cycle.

Reply:
will be updated.